Sources/Plasma/PubUtilLib/plPhysical/plCollisionDetector.cpp

```diff
@@ -684,11 +684,11 @@ bool plPanicLinkRegion::MsgReceive(plMessage* msg)
                     if (avMod->GetController())
                     {
                         avMod->GetController()->GetPositionSim(pos);
-                        DetectorLogSpecial("Avatar is panic linking. Position %f,%f,%f and is %s", pos.fX, pos.fY, pos.fZ, avMod->GetController()->IsEnabled() ? "enabled" : "disabled");
+                        plDetectorLog::Special("Avatar is panic linking. Position {f},{f},{f} and is {f}", pos.fX, pos.fY, pos.fZ, avMod->GetController()->IsEnabled() ? "enabled" : "disabled");
```

Comment:
The last one should be a string, not a float ;)

Reply:
You don't like floating-point strings? :trollface: